x-pack/plugins/security_solution/cypress/downloads/rules_export.ndjson

```diff
@@ -0,0 +1,2 @@
+{"id":"046a74a0-f09d-11ec-abeb-b306649c5f10","updated_at":"2022-06-20T13:29:31.635Z","updated_by":"elastic","created_at":"2022-06-20T13:29:31.635Z","created_by":"elastic","name":"New Rule Test","tags":[],"interval":"100m","enabled":false,"description":"The new rule description.","risk_score":17,"severity":"high","output_index":".siem-signals-default","author":[],"false_positives":[],"from":"now-50000h","rule_id":"rule_testing","max_signals":100,"risk_score_mapping":[],"severity_mapping":[],"threat":[],"to":"now","references":[],"version":1,"exceptions_list":[],"immutable":false,"related_integrations":[],"required_fields":[],"setup":"","type":"query","language":"kuery","index":["apm-*-transaction*","auditbeat-*","endgame-*","filebeat-*","logs-*","packetbeat-*","traces-apm*","winlogbeat-*","-*elastic-cloud-logs-*"],"query":"host.name: *","throttle":"no_actions","actions":[]}
+{"exported_count":1,"exported_rules_count":1,"missing_rules":[],"missing_rules_count":0,"exported_exception_list_count":0,"exported_exception_list_item_count":0,"missing_exception_list_item_count":0,"missing_exception_list_items":[],"missing_exception_lists":[],"missing_exception_lists_count":0}
```

Comment:
Not seeing any references of this being used as input anywhere -- is this just the saved output of a test export? Do we need this in source control?

If it is being used as input and I've missed where, I was just going to mention that we have some rule ndjson's floating around over in [/x-pack/plugins/security_solution/cypress/fixtures](https://github.com/elastic/kibana/tree/main/x-pack/plugins/security_solution/cypress/fixtures) already in case any of those are of use here.

Reply:
Oh, yeah, I missed that. Files in `/cypress/downloads` are just side effects of test runs and not used anywhere. So I'll remove them and add the downloads folder to git ignore.